avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1662,6 +1673,33 @@ function Set-UsageExamplesSection {
                     '<p>'
                 )
             }
+
+            # --------------------- #
+            #   Add Bicep parameters file example   #
+            # --------------------- #
+            if ($addBicepParametersfile) {
+
+                $formattedBicepParametersfileExample = @(
+                    "using 'br/public:$($brLink):$($targetVersion)'"
+                    ''
+                ) + $bicepParametersfileExample
+
+
+                # Build result
+                $testFilesContent += @(
+                    '',
+                    '<details>'
+                    ''
+                    '<summary>via Bicep parameters file</summary>'
```

Comment:
Aligned with the JSON Parameter file section
```suggestion
                    '<summary>via Bicep Parameter file</summary>'
```

Reply:
As discussed, I'll leave it as it is.

Reply:
In this case please align the JSON section. They should both have the same convention

Reply:
was updated @eriqua. Will resolve